features/xacml/org.wso2.carbon.identity.xacml.server.feature/resources/entitlement.properties

```diff
@@ -57,9 +57,9 @@ PAP.Policy.Publisher.Module.1=org.wso2.carbon.identity.entitlement.policy.publis
 #PAP.Policy.Publisher.Verification.Handler=
 PAP.Status.Data.Handler.1=org.wso2.carbon.identity.entitlement.SimplePAPStatusDataHandler
 
-PDP.Policy.Finder.1=org.wso2.carbon.identity.entitlement.dao.RegistryPolicyDAOImpl
+PDP.Policy.Finder.1=org.wso2.carbon.identity.entitlement.dao.RegistryPolicyPersistenceManager
```

Comment:
Having this as registry would lead to use registry impl in a fresh setup right? Shall we have db impl as the default value here?